pkg/operator/rbac.go

```diff
@@ -29,8 +29,9 @@ func MakeRBACObjects(name, namespace, component string, additionalRules []rbacv1
 
 	sa := corev1.ServiceAccount{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      name,
-			Namespace: namespace,
+			Name:        name,
+			Namespace:   namespace,
+			Annotations: fbAnnotations,
```

Comment:
If the user does not set the `serviceAccountAnnotations` field, then this field will be empty, is this case considered?

Reply:
It's ok to set an empty annotation, right? @wanjunlei

Reply:
Yes